### DISCUSSION TITLE
New release

### DISCUSSION THREAD

**Discussion by MariaB99** [SubtitleEdit/subtitleedit#9049] [Q&A]
Hi,

As quite long time passed since the last release and many code fixes/updates where made - I am really interested when new realease will be officially published? 

Thanks for your help

**Comment by niksedk** [comment]
I hope for a release here in December :)

Do give the latest beta a test run: https://github.com/SubtitleEdit/subtitleedit/releases/download/4.0.8/SubtitleEditBeta.zip

**Reply by roshannzth** [reply]
just wanted to check if we can have some automated github actions configured.
lot of bug fixes are done, i guess we should plan for a release soon.
@niksedk

**Reply by niksedk** [reply]
SE 4.0.14 should be out very soon :)
https://github.com/SubtitleEdit/subtitleedit/issues/10006
